index.html

```diff
@@ -63,5 +63,43 @@ <h1>
     
         </div>
     </section>
+
+   <!-- ------------------------------------mobile section-------------------- -->
+<section class="section-nav">
+
+   <header class="header-mobile">
+    <nav>
+        <img src="images/section_header.png" alt="logo">
+    </nav>
+   </header>
+
+<section class="section-mobile">
+   <div class="project-title">
+    <h1>Tonic</h1>
+    <ul class="top-list">
+        <li>CANOPY</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+    </ul>
+   </div>
+    <p>
+        A daily selection of privately personalized 
+        reads; no accounts or 
+        sign-ups required.
+    </p>
+
+    <ul class="list-bottom">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <div class="action">
+      <button>See Project</button>
```

Comment:
- [ ] According to your Figma template, all buttons in your project should implement the following styles:
![image](https://user-images.githubusercontent.com/52635378/170833532-7177cc60-e8fb-46fa-b01d-f0f97fad0c6b.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: